webpack.config.js

```diff
@@ -135,3 +147,14 @@ module.exports = (env, { mode }) => {
     target: ['web', 'es5'],
   };
 };
+
+async function createJavascriptUiFrameworkFilesWithReplacedPrefix() {
+  await exec('npx tsc'); // run the typescript compiler to generate the dist-tsc folder
+  const options = {
+    templatingOn: false, // disable liquid syntax to avoid issues with {{ and }}
```

Comment:
Is there a way to use this built-in feature for this kind of replacement? But I assume that we use this approach on other places as well it could cause some issues there 🤔

Reply:
Do you mean the liquid templating? Given our very little use of templates, I felt it's not worth looking into this further.